setup.py

```diff
@@ -58,7 +58,7 @@ def get_packages(package):
         'Jinja2>=2.10.1',
         'livereload>=2.5.1',
         'lunr[languages]>=0.5.2',
-        'Markdown>=3.0.1',
+        'Markdown>=3.2',
```

Comment:
Note that in addition to here, the following files also need to be updated with this dependency restriction: 

* requirements/project.txt
* requirements/project-min.txt

The tests rely on those.